js/scale-button.js

```diff
@@ -0,0 +1,43 @@
+const imageUploadScale = document.querySelector('.img-upload__scale');
+const scaleSmallerButton = imageUploadScale.children[0];
```

Comment:
Давай здесь все таки через querySelector найдем эти три элемента?
Так мы будем чуть больше уверены, что, если по каким-то причинам разметка изменится, то js не сломается.

```suggestion
const scaleSmallerButton = imageUploadScale.querySelector('.scale__control--smaller');
const scaleInputValue = ...
const scaleBiggerButton = ...

```